### PR TITLE
Adding dockerfile because idk maybe we will need it

### PR DESCRIPTION
**Pull request by btmoore4**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM node:12
```
Comment (dangalan) [inline Dockerfile:1]: Maybe bump this to a more recent version? Looking at their website I think 14 or 15 is the latest stable version

**Comment by dangalan** [comment]
Maybe we could also add a `.dockerignore` file for the `node_modules` folder since the Dockerfile runs `npm install`? Not sure what's best practice though

File: Dockerfile (hunk 1 above)
Comment (btmoore4) [reply Dockerfile]: Done 🌮

**Comment by btmoore4** [comment]
> Maybe we could also add a `.dockerignore` file for the `node_modules` folder since the Dockerfile runs `npm install`? Not sure what's best practice though

Looked it up, looks like that is the best practice

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.node_modules
```
Comment (dangalan) [inline .dockerignore:1]: I think this should be just `node_modules` without the `.` or `**/node_modules` which I believe should match that folder name in any directory according to the [Docker docs](https://docs.docker.com/engine/reference/builder/)

**Review by dangalan — APPROVED** [review]

**Pull request merged**